### PR TITLE
Adds the articles section

### PR DESCRIPTION
**Pull request by chineduknight**
#### What does this PR achieve?
- Adds the Articles Section
- Uses styles from mockup

#### Screenshots:

articulet

#### Relevant Trello Tickets
- [Articulte Section](https://trello.com/c/gCgPhO8D/7-article-section)

### CODE REVIEW

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import HeadingText from 'Atoms/HeadingText';
+import Image from 'Atoms/Image';
+import __fontSizes from 'FontSizes';
+import __devices from 'Devices';
+
+const ArticulesSection = () => {
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]: Change this to use `function declaration. Also, you made an error in the spelling of **Articles**
```suggestion
function ArticlesSection{
```

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  const ROBOT_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117293/Ojumo/2EJCSULRwC8.png';
+  const CHILDREN_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117292/Ojumo/AEaTUnvneik.png';
+  const articules = [
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]:
```suggestion
  const articles = [
```

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    },
+    {
+      imageURL: CHILDREN_IMAGE,
+      title: 'Extracts from our visit to the ophanage',
+      body:
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, modi quod dolorem cumque iure totam blanditiis accusantium amet quia facilis ipsum distinctio tempora? Soluta.',
+      date: 'Date - 20th May 2019',
+      time: 'Time - 8am - 4pm'
+    }
+  ];
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]: I think this array should have the keys:
- imageURL
- title
- author
- datePublished
- description

review the mocks to see how these fields are used

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    align-items: center;
+    letter-spacing: 0.1em;
+   
+  }
+
+  .read-more {
+    font-family: Lato;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13px;
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]: as nice as fixed measurements are, using relative measurements would make your responsiveness better. You can quickly convert to relative measurement by dividing the fixed measurement by `16` (ie `pxValue / 16`). 

Also, check the `font_sizes` file to check for existing relative measurements

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+  h1 {
+    font-family: Montserrat;
+    font-style: normal;
+    font-weight: 800;
+    font-size: ${__fontSizes.point1_5};
+    line-height: 24px;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    margin: 20px 0;
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]: see fixed measurement feedback

File: src/components/layouts/landingPage/ArticuleSection/index.jsx [hunk 6]
```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import HeadingText from 'Atoms/HeadingText';
+import Image from 'Atoms/Image';
+import __fontSizes from 'FontSizes';
+import __devices from 'Devices';
```
Comment (chidioguejiofor) [inline src/components/layouts/landingPage/ArticuleSection/index.jsx]: These imports using path aliases just look clean 👍

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
I see the new section. Looks like what we are trying to achieve( almost 😉 ). I found that the UI is not responsive in the following screenshots

Screenshot 2020-01-28 at 1 55 39 PM
Screenshot 2020-01-28 at 1 54 26 PM
Screenshot 2020-01-28 at 1 53 40 PM

The last 2 screenshots show a misalignment but here is how it is from the mock
Screenshot 2020-01-28 at 1 57 53 PM

I think using relative measurements wherever possible could fix this. Else we call on the good old `media queries`

Also dropped some feedback

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
Also, I noticed that you did not follow the commit conventions and you have 2 commits in this PR. Please work on those as well

File: src/components/layouts/landingPage/ArticuleSection/index.jsx (hunk 3 above)
Comment (chineduknight) [reply src/components/layouts/landingPage/ArticuleSection/index.jsx]: can't find where this is specified in the mocks
I have updated but can't find where author is used

**Comment by chineduknight** [comment]
> n

I not clear on this convention

File: src/components/layouts/landingPage/ArticuleSection/index.jsx (hunk 1 above)
Comment (chineduknight) [reply src/components/layouts/landingPage/ArticuleSection/index.jsx]: Is there a reason for this

File: src/components/layouts/landingPage/ArticuleSection/index.jsx (hunk 2 above)
Comment (chineduknight) [reply src/components/layouts/landingPage/ArticuleSection/index.jsx]: updated

File: src/components/layouts/landingPage/ArticuleSection/index.jsx (hunk 4 above)
Comment (chineduknight) [reply src/components/layouts/landingPage/ArticuleSection/index.jsx]: updated

File: src/components/layouts/landingPage/ArticuleSection/index.jsx (hunk 5 above)
Comment (chineduknight) [reply src/components/layouts/landingPage/ArticuleSection/index.jsx]: I don't understand this feedback?
Should I use the font-size fixed measurement for the margin and line spacing or should I just convert pixels to rem?

I currently converted to rem

File: src/components/pages/LandingPage/index.jsx [hunk 7]
```diff
@@ -3,13 +3,15 @@ import OjumoBriefInfo from '../../layouts/landingPage/OjumoBriefInfo'
 import MissionSection from '../../layouts/landingPage/MissionSection'
 import Footer from '../../layouts/landingPage/Footer'
 import UpcomingEvents from '../../layouts/landingPage/UpcomingEvents'
+import ArticulesSection from '../../layouts/landingPage/ArticuleSection'
```
Comment (chidioguejiofor) [inline src/components/pages/LandingPage/index.jsx]: type on `ArticulesSection`:

```suggestion
import ArticlesSection from '../../layouts/landingPage/ArticleSection'
```

File: src/components/pages/LandingPage/index.jsx [hunk 8]
```diff
@@ -3,13 +3,15 @@ import OjumoBriefInfo from '../../layouts/landingPage/OjumoBriefInfo'
...
 export default function LandingPage(){
 
     return (
         <div>
               <OjumoBriefInfo/>
               <MissionSection/>
               <UpcomingEvents/>
+              <ArticulesSection/>
```
Comment (chidioguejiofor) [inline src/components/pages/LandingPage/index.jsx]:
```suggestion
              <ArticlesSection/>
```

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
So there's this misalignment between the two articles card on desktop view. 

Screenshot 2020-02-13 at 7 07 21 PM

It would be nice if you could look at that.

The key thing holding this PR though is the number of commits(there are 3 of them). I would like you to squash them into one. If you don't know how to do this I can help over a call.

File: src/components/pages/LandingPage/index.jsx (hunk 8 above)
Comment (chidioguejiofor) [reply src/components/pages/LandingPage/index.jsx]: No change to this

**Comment by chidioguejiofor** [comment]
I just fixed
Screenshot 2020-02-25 at 10 19 16 AM
 all pending issues

**Pull request merged**